internal/cli/cli.go

```diff
@@ -66,20 +72,24 @@ func Lookup(name string, commands []*Command) (*Command, error) {
 // SetFlags registers a list of functions that configure flags for the command.
 func (c *Command) SetFlags(flagFunctions []func(fs *flag.FlagSet)) {
 	c.flags = flag.NewFlagSet(c.Name, flag.ContinueOnError)
-	c.flags.Usage = constructUsage(c.flags, c.Name)
+	c.flags.Usage = c.usage()
 	for _, fn := range flagFunctions {
 		fn(c.flags)
 	}
 }
 
-// TODO(https://github.com/googleapis/librarian/issues/205): clean up this
-// function so that "librarian" is not hardcoded
-func constructUsage(fs *flag.FlagSet, name string) func() {
-	output := fmt.Sprintf("Usage:\n\n  librarian %s [arguments]\n", name)
-	output += "\nFlags:\n\n"
+func (c *Command) usage() func() {
+	if c.Name == "" || c.Short == "" || c.Usage == "" || c.Long == "" {
+		panic(fmt.Sprintf("command %q is missing documentation", c.Name))
+	}
+
+	output := fmt.Sprintf("Name:\n  %s\n\n", c.Name)
+	output += fmt.Sprintf("Usage:\n  %s\n\n", c.Usage)
+	output += fmt.Sprintf("Description:\n  %s\n", c.Long)
```

Comment:
I would imagine that Long will be multiple lines in some cases. Is it worth splitting `Long` on `\n` and then joining with the two-space prefix? (Definitely a nice-to-have that we could do later, of course.)